resources/wikia/modules/localStorage.js

```diff
@@ -5,6 +5,9 @@ define('wikia.localStorage', ['wikia.window'], function(window) {
 	'use strict';
 
 	try {
+		window.localStorage.setItem('localStorageTestItem', 'testValue');
+		window.localStorage.getItem('localStorageTestItem');
```

Comment:
Fair point. It's used by one module, so I'm going to refactor it a little bit and remove `wikia.localStorage`